data/hai/hai reveal 4 middle.txt

```diff
@@ -1456,32 +1458,410 @@ mission "Hai Reveal [A11] Sound the Alarm"
+			`	It warns the growing movement has become a threat to the stability of both Hai and human society.`
+				goto reading
+
+			label quarg
+			action
+				log `A Hai intelligence report reports a mutual fondness between Hai and Quarg for one anothers' cuisine. It also mentioned lingering frustration about the Quarg refusing to fight the Korath during their warlike spiral that led to near extinction. The Hai felt that Quarg involvement would have saved more Korath lives.`
+				set "Hai Reveal [A11] Sound the Alarm: quarg book"
+				"Hai Reveal [A11] Sound the Alarm: time" ++
+			`	This report is surprisingly light on details of any interest. It seems the Hai view the Quarg as a nearby race of friendly neighbors. There's frequent cultural exchanges, and a large food trade in both directions. Apparently, "genuine Hai cuisine" is popular among the Quarg, especially the famous Hai acorn cookies. In return, the Hai use Quarg spices extensively, including some of the meals you've eaten since you arrived. The spices are described as "subtle and mysterious."`
+			`	Between the mundane and cultural, you find a few interesting facts. There is some lingering frustration about the Quarg refusing to help fend off Korath raids. This is partly due to the Hai having to fight for themselves, but mostly out of a belief that more Korath would have survived the war if the Quarg had intervened. Also, the Hai apparently have no interest in developing the jump drives Quarg have in abundance due to "the fate of the Korath," and warnings from the Quarg about the dangers of the jump drive manufacturing process.`
+				goto reading
+
+
+			label drak
+			action
+				set "Hai Reveal [A11] Sound the Alarm: drak book"
+				"Hai Reveal [A11] Sound the Alarm: time" ++
+			`	Although this report is almost a hundred pages long, there's only about three pages of text. It seems the Hai have no certain knowledge about the Drak, save that they're friends of the Quarg. There are legends in the Mythos of the Dara-Irik, faceless creatures of destruction, but the report notes the similarity in name is likely a coincidence. It also describes "Drak Archons," giant space creatures, whom the Hai credit both for saving the Korath and ending the raids on the Hai. However, they're uncertain about whether the Archons are Drak, are ships that contain Drak, or are merely sentient creations of the Drak.`
```

Comment:
> It also describes "Drak Archons," giant space creatures, whom the Hai credit both for saving the Korath and ending the raids on the Hai.

Something about this is off but I can't think of a way to replace it.

Reply:
I rewrote the report and added the missing log entry. How does it look now?

Reply:
I should have highlighted that it was really `whom the Hai credit both for saving the Korath and ending the raids on the Hai` that sounded weird. Whose raids on the Hai?

Reply:
Ah, yes, that is a critical detail, isn't it? Now it clarifies that the Archons ended the Korath raids on the Hai.

Reply:
> Ah, yes, that is a critical detail, isn't it? Now it clarifies that the Archons ended the Korath raids on the Hai.

That is new lore that has never thus far been alluded to, and would also substantially undermine the merits of the Unfettered's position.

Reply:
~~thx for telling me~~
It probably is what happened, to be fair, and explains why the war ended all of a sudden with the proper ships to respond to the raids not mass produced yet.
But that does not mean some Unfettered won't believe it's thanks to them the raids stopped - or if it was someone else

I think we can go with the idea it was a mystery - I don't see the Archons showing up like "hey we stopped the raids, bye now have a nice day" so they probably learned it from the Quarg, eventually, but not all choose to believe it?

Reply:
No, no, no. The information in the reports is confidential, and comes from Hai Intelligence. They are aware that the Archons stopped the Korath mass suicide, and that the Korath stopped raiding not long after. It isn't difficult for an intelligence agency to establish a causal relationship.

That does *not* mean that the Unfettered have come to the same conclusion, nor even that the Hai public is aware of the Archons.

Reply:
I have updated the intelligence report to clarify it is providing the knowledge of Hai Intelligence, not the Unfettered or the Hai public.

Reply:
> Also, the Hai had access to plenty of jump drives from the Korath ships raiding them.

do they? Maybe a few but if they did they would not risk them (unlike the Unfettered), and this was never established either

Reply:
I can remove the Drak intelligence report if absolutely necessary, but I'd rather leave it in, to fill in the plot holes. I am *not* adding anything to the lore in that report; it's simply inescapable conclusions from available information: the Hai are not morons; if they asked the Quarg what happened to the Korath, they would have gotten an answer; and their military (the Unfettered) *would* have pillaged and use jump drives to investigated. From all of this, it is impossible for the Hai not to know what happened to the Korath. There will still be enough intelligence reports for the mission if I remove this one, though the buffer is getting quite thin.

Reply:
I removed the Drak intelligence report since I don't feel like arguing about it anymore. There's still one extra report, so the buffer is wearing quite thin.